packages/javascript-sdk/package.json

```diff
@@ -21,7 +21,6 @@
   },
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk#readme",
   "devDependencies": {
-    "rollup-plugin-terser": "^7.0.2",
```

Comment:
I see this was removed, but I can't find any corresponding code change. Was this never needed?